packages/store/src/scalarMap.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2019 Agoric, under Apache license 2.0
```

Comment:
I don't think this copyright is necessary

Reply:
Gone